config/env.ts

```diff
@@ -37,6 +37,12 @@ export const WP_BASIC_AUTH_PASSWORD = checkEnv(
 
 function checkEnv(env: string | null | undefined, envName: string): string {
    if (env == null) {
+      if (process.browser) {
+         if (envName.startsWith('NEXT_PUBLIC')) {
+            console.warn(`environment variable "${envName}" is not defined`);
```

Comment:
I'm confused why we are emitting a warning that says '`envName` is not defined' when we know envName is a string, and we are checking that it begins with 'NEXT_PUBLIC'. Perhaps this was a copy/paste mistake?

Reply:
@masonmcelvain `envName` is the name of the environment variable, the null check is actually on the `env` value as you can see in line 39.

To be clear, the purpose of this function is to output clear errors when an environment variable is missing. 
When this is used server side we throw an error, so it fails the tests. 
When this is used client side we need to distinguish between variables that should be available client side (those starting with `NEXT_PUBLIC`) and variables that are available only server side. In the former case we should output at least a warning if the variable is not defined, in the latter case we should not output anything.

Reply:
Just saw this- thanks for the explanation, that makes perfect sense :)